src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -733,10 +749,37 @@ internal async Task SearchPackagesAndRefreshUpdateCountAsync(string searchText,
                 loadContext.CachedPackages = Model.CachedUpdates;
             }
 
-            var packageFeed = await CreatePackageFeedAsync(loadContext, _topPanel.Filter, _uiLogger);
+            // only make recommendations when
+            //   the single source repository is nuget.org,
+            //   the package manager was opened for a project, not a solution,
+            //   this is the Browse tab,
+            //   and the search text is an empty string
+            if (loadContext.SourceRepositories.Count() == 1
+                && loadContext.SourceRepositories.First().PackageSource.Source.ToLower() == "https://api.nuget.org/v3/index.json"
+                && loadContext.IsSolution == false
```

Comment:
I would switch the ordering here. 
You want the cheapest check to be first.

Reply:
Fixed

Reply:
Moved the .Count() condition after all of the other == conditions.